### ISSUE TITLE
Missing suggested answers / example answers

### ISSUE THREAD

**Issue by sjDCC** [DMPRoadmap/roadmap#334]
I can't see how to get example or suggested answers to display. I know there are some on the St Andrew's customisation of the EPSRC template. I have changed my institutional affiliation to St Andrew's and created a plan for EPSRC. I then selected the St Andrew's guidance as this doesn't auto-select (see issue #326), but even then the suggested answers still don't display. 

This is what shows on the live site
![suggested-answer](https://cloud.githubusercontent.com/assets/6169940/26287478/454327a0-3e74-11e7-92af-ee427c35e9b2.PNG)

And what shows on test
![missing-suggested-answer](https://cloud.githubusercontent.com/assets/6169940/26287479/49182952-3e74-11e7-9a80-fc05eacf5836.PNG)

It's seems we're not pulling in any institutional customisations of templates, so I expect any additional sections won't show either. Will look for examples to test that theory.

**Comment by sjDCC** [comment]
Having checked for additional sections, I have found that these are displaying without guidance being picked. The test I did was for Sheffield Hallam which has questions added to the H2020 template. These showed in the template on test (see below) so it seems the customisations are being picked up to some extent. This suggestions sections are coming in, but earlier tickets indicate that guidance by question and suggested answers are not (#341, #335)
![section-test](https://cloud.githubusercontent.com/assets/6169940/26288063/e3f27e1c-3e80-11e7-96fc-71ee4a1b6d3f.PNG)

The guidance from SHU in the EC template is mis-tagged though. See comparison screengrabs below. Perhaps we have a blanket naming for all guidance within a funder template and this is overriding question annotations from others?
![shu-template-in-ec-template-live](https://cloud.githubusercontent.com/assets/6169940/26288083/9adb36aa-3e81-11e7-9f84-a6b2a99d5131.PNG)
![shu-guidance-in-ec-template-test](https://cloud.githubusercontent.com/assets/6169940/26288084/9fd59448-3e81-11e7-8e2c-eab1811e54af.PNG)

This test uncovered another bug. The EC template that is being used to create plans on the test site is the old unpublished version. Will raise this in another ticket.
![shu-template-in-ec-template-live](https://cloud.githubusercontent.com/assets/6169940/26288081/8b9793c8-3e81-11e7-8de0-0194f9a3d30a.PNG)

**Comment by vyruss** [comment]
cause identified and @xsrust is working on issue.

**Comment by vyruss** [comment]
Related to #339

**Comment by sjDCC** [comment]
This still seems to be broken. Suggested answers aren't displaying, noted in issue #372

**Comment by xsrust** [comment]
This should no longer be blocked by #372

**Comment by xsrust** [comment]
Contents of this ticket contained within other tickets
